packages/native/types/base.d.ts

```diff
@@ -73,9 +69,11 @@ type ReactNativeTags =
   | 'ViewPagerAndroid'
   | 'WebView'
 
-export type ReactNativeElements = {
-  [Tag in ReactNativeTags]: PropsOf<ReactNative[Tag]>
-}
+export type ReactNativeComponents = Pick<ReactNative, ReactNativeComponentNames>
```

Comment:
`ReactNativeElements` is a very popular `react-native` design system, so to be less confusing I renamed this to a more descriptive `ReactNativeComponents`. Additionally, converting the mapped type description into the more compact (and equivalent) `Pick` construct.